config/src/main/kotlin/org/springframework/security/config/annotation/web/FormLoginDsl.kt

```diff
@@ -51,6 +51,15 @@ class FormLoginDsl {
 
     private var defaultSuccessUrlOption: Pair<String, Boolean>? = null
 
+    private var disabled = false
+
+    /**
+     * Disable FormLogin.
```

Comment:
Will you please add a `@since` tag here? e.g. `@since 6.1`

Reply:
Makes sense